src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -176,6 +176,7 @@ def load_arguments(self, _):
 
     with self.argument_context('functionapp list-runtimes') as c:
         c.argument('os_type', options_list=["--os", "--os-type"], help="limit the output to just windows or linux runtimes", arg_type=get_enum_type([LINUX_OS_NAME, WINDOWS_OS_NAME]))
+        c.argument('sku', help="list runtime stacks for this specific sku. Only 'flex' sku is supported for this parameter. Use --os for other SKUs.", arg_type=get_enum_type(['flex']))
```

Comment:
This would make the `--sku` param only accept `flex`. We need to have an option for the other skus as well. Not sure what would be the best way to represent it for customers. Might need to ask Shreya or Thiago. Maybe something like ['flex', 'any'] would suffice?

Reply:
Will talk to PMs about it.